lib/twine/cli.rb

```diff
@@ -11,7 +11,7 @@ def self.parse_args(args, options)
       new(args, options).parse_args
     end
 
-    def parse_args
+    def parse_args    # TODO: let this method take args and return options
```

Comment:
Up to you. I sometimes put smaller things like that on my own TODO list. It really depends on how important you think it is.